submissions/oluwabusolamiEluyela/html-forms/contact.html

```diff
@@ -5,14 +5,19 @@
     <title>Contact Us!</title>
 </head>
 <body>
-<nav>
+<header>
     <h1>JUSTIFY</h1>
+    <nav>
     <a href="../html/index.html">Product Page</a>
     <a href="../html-forms/Contact.html">Contact Us</a>
-</nav>
+    </nav>
+</header>
+<section>
+  <div>
     <h1>Contact Form</h1>
-    <form action="#" method="post" enctype="multipart/form-data">
```

Comment:
Is there a reason why you are removing the opening `<form>` tag?

Reply:
It was a mistake

Reply:
You have two opening form tag

Reply:
Kindly go through your code